.changeset/strong-eyes-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/discussion-rendering': patch
```

Comment:
if we're adding a new required property to the app, should this be a major change instead?

Reply:
Yes, good point! I have updated the changeset.

Reply:
thanks!